apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowAsyncEnumerator.cs

```diff
@@ -33,6 +33,7 @@ namespace Google.Cloud.Bigtable.V2
     internal sealed class RowAsyncEnumerator : IAsyncEnumerator<Row>
     {
         private CellInfo _currentCell;
+        private SortedList<string, Family> _currentFamilies = new SortedList<string, Family>(StringComparer.Ordinal);
```

Comment:
Could this be readonly? It looks like it could.

Reply:
Yup, done.